roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -31,6 +31,9 @@
       loop:
         - iptables.service
         - ip6tables.service
+      when:
```

Comment:
Not sure if having only iptables without IPv6 is possible.
If it is, this could be better:
```
when:
  - ansible_facts.services["{{ item }}"] is defined
```

Reply:
Se the below reply :) 

BTW using item it's better, I'll fix it ASAP